src/new-components/heroes/image-hero.tsx

```diff
@@ -0,0 +1,46 @@
+import { GatsbyImage, getImage, IGatsbyImageData } from 'gatsby-plugin-image';
+import React from 'react';
+import { HeroContainer, TextContainer } from './support';
+import { HeroText } from './hero-text';
+import { HeroWithText } from './types';
+
+const coverContainerCss = {
+  gridArea: '1/1',
+  /**
+   * make sure the image, which is positioned absolute by gatsby,
+   * is behind any content on the current stacking context.
+   */
+  zIndex: -1,
+};
+
+type ImageHeroProps = Partial<HeroWithText> & {
+  image: IGatsbyImageData;
+};
+
+/**
+ * Display any given gatsby image as a hero image.
+ */
+export const ImageHero = ({
+  image,
+  title,
+  children,
+  kicker,
+}: ImageHeroProps) => {
+  const gatsbyImageData = getImage(image);
+
+  return (
+    <HeroContainer>
```

Comment:
Welcome `ImageHero` 🌟 You will save so much brain power in the future because you should just work